lib/saxerator/parser/accumulator.rb

```diff
@@ -5,28 +5,27 @@ def initialize(config, block)
         @stack = []
         @config = config
         @block = block
-        @builder = Builder.to_class(@config.output_type)
       end
 
       def start_element(name, attrs = [])
-        @stack.push @builder.new(@config, name, Hash[*attrs.flatten])
+        @stack.push @config.output_type.new(@config, name, Hash[attrs])
       end
 
       def end_element(_)
-        if @stack.size > 1
+        if @stack.length > 1
```

Comment:
@soulcutter I've updated it acording to https://github.com/JuanitoFatas/fast-ruby#array